### ISSUE TITLE
Minor Bug in artist's main pages

### ISSUE THREAD

**Issue by Ali-Mohamed-Abdelmawla** [SpotX-Official/SpotX#643]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Egypt

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.1.90.859.gf1bb1e36
- Windows: 11
- PowerShell: 5.1.19041.1200
```

### 📝 Description

The issue persists in the any artist's main page with the show more && show less button in the best songs list for the artist
When i click on the button to show more it is perfectly fine and displays to you the 10 most popular songs for the artist 
 but the bug exists upon collapsing the list again to 5 songs

**Comment by amd64fox** [comment]
apparently you specified the incorrect client version, I didn't find such a bug in 1.1.90, but I found it in the latest versions
1.2.46 +

**Comment by Ali-Mohamed-Abdelmawla** [comment]
sorry i didn't get the version right 
i just wanted to point out to the issue

**Comment by amd64fox** [comment]
fixed in version 1.2.49

for older versions I will make a patch later
